libraries/ui-shared/Dialog/components/PipelineErrorDialog/index.jsx

```diff
@@ -99,32 +100,65 @@ class PipelineErrorDialog extends Component {
 
   /**
    * Renders the error message in developer mode.
-   * @param {Object} params The message parameters.
    * @returns {JSX}
    */
-  renderDevErrorMessage = params => (
-    <div aria-hidden>
-      <strong>Pipeline:</strong> {params.pipeline}<br />
-      { params.code && (
-        <span>
-          <strong>Code: </strong> {params.code} <br />
-        </span>
-      )}
-      <strong>Message:</strong> {params.message} <br />
-      <p>
-        <strong>Params:</strong><br />
-        {JSON.stringify(params.request, null, ' ')}
-      </p>
-    </div>
-  );
+  renderDevErrorMessage = () => {
+    const { params } = this.props;
+
+    return (
```

Comment:
This is invalid JSX code! Please check your IDE. eslint should have complained about this, as it does for me!
If you want to render whitespaces at the beginning and the end of a string you should use template strings.
Also, all these unnecessary whitespaces should be removed and your IDE should've complained.

Bad:
```
{ params.entityId && (
  <span>
    <strong>Entity id: </strong> {params.entityId}<br />
  </span>
)}
```
Good:
```
{params.entityId && (
  <span>
    <strong>{`Entity id: `}</strong>
    <span>{` ${params.entityId}`}</span>
  </span>
)}
```

Reply:
Thanks for the advice. Didn't notice such a behavior.

Reply:
I had one space too much before the `</strong>` tag and will correct that one. But the other spaces are fine and show up correctly in the app as expected. Why would I need to use template strings when React renders it correctly?
I discovered an other display issue when the properties get a value of `0`, which I will fix as well.
Regarding eslint: My IDE is set up correctly as far as I can tell. I ran eslint on the command line and there I don't see any problems either.
Can you please tell me the name of the eslint rule, which is your IDE complains about?